setup.py

```diff
@@ -75,6 +75,7 @@ def run(self, *arg, **kw):
     'pymongo>=3.5',
     'PyYAML',
     'psutil',
+    'pyOpenSSL',
```

Comment:
Why is pyOpenSSL needed here?

Reply:
Circle is failing with `pkg_resources.DistributionNotFound: The 'pyOpenSSL>=0.14; extra == "security"' distribution was not found and is required by requests`.  We specify pyOpenSSL explicitly in master's setup.py, so I thought this was the most harmonious way to update the 2.x-maintenance branch.

Reply:
Does `requests[security]` bring this in? If so, would it be better just to specify that?

Reply:
It would not surprise me if it didn't.  There are all sorts of edge cases related to pip resolving extra requirements that are requested through `install_requires`.  This is probably why it ended up in master.

Reply:
I'll move this to its own PR.